package.json

```diff
@@ -32,6 +34,7 @@
   "devDependencies": {
     "@types/node": "latest",
     "@types/jest": "latest",
-    "jest": "latest"
+    "jest": "latest",
+    "typescript": "latest"
```

Comment:
That's ok, I'll remove it. I don't use a global typescript lib because more often than not you need to work with different versions of typescript per project. That's why I added it.